micronaut-maven-integration-tests/src/it/package-docker-native-static/selector.groovy

```diff
@@ -0,0 +1,6 @@
+return isX86()
+
+static boolean isX86() {
```

Comment:
This seems to be used nowhere?

Reply:
It's a [selector script](https://maven.apache.org/plugins/maven-invoker-plugin/examples/selector-scripts.html)